### ISSUE TITLE
Support custom IP address range

### ISSUE THREAD

**Issue by agners** [meatpiHQ/wican-fw#16]
When in AP mode, the device uses 192.168.80.x as address range. It would be helpful to have an option to change the address range used, in case that address range happens to be used :sweat_smile:

**Comment by meatpiHQ** [comment]
@agners When you're connected to WiCAN AP your device gets an IP from WiCAN dhcp something like 192.168.80.2. Are you connected to WiCAN AP and another network at the same time?

**Comment by agners** [comment]
> Are you connected to WiCAN AP and another network at the same time?

Yes, that is what I would like todo, but I get routing issues because I use the same range. Reconfigure my whole network is a bit painful :cold_face:

**Comment by meatpiHQ** [comment]
@agners If you send me the IP range you want I can give you a hardcoded version to get you going, and I'll add this option in the next release.

**Comment by agners** [comment]
Thanks for the offer! I can manage for now. An option in the next release is fine.
